xml/ay_create_control_file.xml

```diff
@@ -297,4 +297,83 @@
     <link xlink:href="http://www.w3.org/TR/xslt">www.w3.org/TR/xslt</link>
    </para>
   </sect1>
+
+  <sect1 xml:id="check-profile">
+   <title>Checking a Control File</title>
+
+   <para>
+    Depending on the use case, creating an &ay; profile can be difficult,
+    especially if you build a dynamic one using <link
+    xlink:href="part-dynamic-profiles">rules/classes, ERB templates or
+    pre-scripts</link>. To simplify the testing and debugging process, &ay;
+    offers a <literal>check-profile</literal> command that takes care of
+    fetching, building and, optionally, importing the profile to detect any
+    potential problem.
+   </para>
+
+   <sect2>
+    <title>Basic Checks</title>
+
+    <para>
+     The simplest way to use this command is just to read and validate the profile:
+    </para>
+
+<screen>yast2 autoyast check-profile filename=autoinst.xml output=result.xml</screen>
+
+    <para>
+     The <filename>result.xml</filename> file contains the result of evaluating
+     the profile. Bear in mind that, even if you do not use any advanced
+     feature, the content of <filename>autoinst.xml</filename> and
+     <filename>result.xml</filename> may differ. The reason is that &ay; does
+     some clean-up when it processes the profile.
+    </para>
+
+    <para>
+     <literal>check-profile</literal> can deal with remote files too:
+    </para>
+
+<screen>yast2 autoyast check-profile filename=http://192.168.1.100/autoinst.xml output=result.xml</screen>
+
+   </sect2>
+
+   <sect2>
+    <title>Running pre-scripts</title>
+
+    <para>
+     Optionally, &ay; is able to run pre-scripts too. In order to enable this feature,
+     you need to set the <literal>run-scripts</literal> to <literal>true</literal>.
+    </para>
+
+<screen>yast2 autoyast check-profile filename=http://192.168.1.100/autoinst.xml output=result.xml run-scripts=true</screen>
+
+    <warning>
+     <para>
+      Needless to say that you must be careful when running pre-installation
+      scripts as root.
+     </para>
+    </warning>
+   </sect2>
+
+   <sect2>
+    <title>Importing the Profile</title>
+
+    <para>
+     It is possible to face some problems when importing a valid profile, even
+     if it is correct. The reason is that AutoYaST does not perform any logic
+     check when fetching, building and validating the profile.
```

Comment:
It is by default enabled, so it is more likely just import-all=false which is needed when you just want resulting profile and not do import checks ( including autoyast issues ). I would also note that it do check on current system with current context, so e.g. different archs or code streams can have different issues or it can contain for them false positives. and same for xml schema check.